sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/CombineFns.java

```diff
@@ -588,6 +597,13 @@ public CoCombineResult extractOutput(Object[] accumulator, Context c) {
       }
       return new ComposedAccumulatorCoder(coders);
     }
+
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      for (CombineFnWithContext combineFn : combineFnWithContexts) {
+        builder.include(combineFn);
```

Comment:
What will happen here if the `CombineFnWithContext` includes the same kind of CombineFn twice? Eg, `Top(5)` and `Top(50)`?